org_fedora_oscap/gui/spokes/oscap.py

```diff
@@ -426,7 +426,7 @@ def _init_after_data_fetch(self, wait_for):
             self._ds_checklists = self._content_handler.get_data_streams_checklists()
             add_ds_ids = GtkActionList()
             add_ds_ids.add_action(self._ds_store.clear)
-            for dstream in self._ds_checklists.iterkeys():
+            for dstream in list(self._ds_checklists.keys()):
```

Comment:
Why is the `list()` function here? In python3, `.keys()` method returns a new view of the dictionary’s keys, which you can iterate over without any further conversions.

Reply:
@jan-cerny this was more of a python 2 to python 3.6+ jump which I believe 3.6 uses/prefers `list(d.keys())` instead of just plain `d.keys()`. Correct me if I am wrong and misunderstanding something here.

Reply:
@jan-cerny I revisted this. What happened was that I had already fixed it correctly, then got distraced and fixed it again to be wrong. Fixed correctly now.